app/helpers/sheet/invoice_list.rb

```diff
@@ -19,16 +16,16 @@ def title
       end
     end
 
+    def parent_sheet
+      create_parent(Sheet::Group)
+    end
+
     def left_nav?
       true
     end
 
     def render_left_nav
-      if form?
```

Comment:
War das nicht aus einem bestimmten Grund hier? Funktioniert jetzt das Erstellen von neuen Einzel- oder Sammelrechnungen auf einer Einzelperson oder Personenliste noch korrekt?

Reply:
Das funktioniert weiterhin korrekt und ich wäre auch sehr erstaunt, wenn das nicht der Fall wäre.

Dieser Handstand war "nötig", weil die falsche Section in der linken Navigation ausgeklappt war. Dadurch erschien es sinnvoll, die Gruppenstruktur anzuzeigen. Ich habe es jetzt so gelöst, dass man bei Rechnungen im Rechnungsbereich ist. So ist die Navigation ruhiger und schlüssiger. Zur relevanten Gruppe kommt man wie immer auf vielen Wegen, insbesondere natürlich das ummantelnde Sheet.